src/components/views/Jobs.js

```diff
@@ -145,24 +147,50 @@ export default class Jobs extends React.Component {
     );
   }
 
+  renderNewJobMenu = ({ onToggle }) => (
+    <DropdownMenu
+      data-role="menu"
+      aria-label={<FormattedMessage id="ui-local-kb-admin.job.availableJobTypes" />}
+      onToggle={onToggle}
+    >
+      <FormattedMessage id="ui-local-kb-admin.job.newJob">
+        {ariaLabel => (
```

Comment:
Couple things:
1. We shouldn't use the same aria-label for two buttons side by side, that's confusing.
2. `aria-label` should only actually be used in these cases where the button's label is ambiguous. Eg, in the export kbart/json dropdown button in ui-agreements, the button's label is just `JSON` or `KBART` because the menu's label is `Export as...`. But in this spot, it looks like the button's label is clear enough and aria-label is just extra fluff that makes the screen reader annoying.

Reply:
I'm not sure what you mean about the two same aria-labels, I can't see the duplicate. I've removed the top aria-label though anyway, as I think that's what you were saying to do here.

Reply:
Ah, ok, aria-label removed fully